Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM python:3.8-buster
```

Comment:
Please add a copyright notice at the top of the file. You can copy it from one of the other files, and simply replace "Google LLC" with your own name.